ckanext/qdes/access/logic/auth/get.py

```diff
@@ -1,10 +1,10 @@
 import ckan.plugins.toolkit as toolkit
 
 
-@toolkit.chained_auth_function
-def api_token_list(next_auth, context, data_dict):
-    # sysadmins only
-    return {'success': False}
+# @toolkit.chained_auth_function
```

Comment:
Can remove the unused code? We can always use the commit history so don't need to comment it out